### PR TITLE
set_my_commands method

### PR DESCRIPTION
**Pull request by CrisMystik**
Added the client.set_my_commands() method as the SetMyCommands botapi function.
* Async version: https://github.com/CrisMystik/pyrogram/tree/pr1-asyncio

### CODE REVIEW

**Comment by delivrance** [comment]
Maybe we should mimic the bot api arguments and types since this is a bot-only feature, but instead of `set/get_my_commands` we should call it `set/get_bot_commands` because it's weird for user identities.

**Comment by delivrance** [comment]
Also, there's no need to duplicate the work for asyncio since currently is mostly a matter of adding async/await keywords where needed (I do this myself when merging from the develop branch).

File: pyrogram/client/methods/users/set_my_commands.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+from pyrogram.api import functions, types
+from ...ext import BaseClient
+
+
+class SetMyCommands(BaseClient):
+    def set_my_commands(
+        self,
+        commands: Dict[str, str]
+    ) -> bool:
+        """Update your bot commands list ("/" icon in official Telegram clients).
```
Comment (JosXa) [inline pyrogram/client/methods/users/set_my_commands.py:30]: > ("/" icon in official Telegram clients)

I'd honestly just leave it out.
If you wanna keep it, maybe shorten to `(slash-commands)` ?

**Review by JosXa — APPROVED** [review]
Nice!

**Pull request closed without merging**